source/adaptive_load/input_variable_setter_impl.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "nighthawk/adaptive_load/input_variable_setter.h"
+
+#include "api/adaptive_load/input_variable_setter_impl.pb.h"
+
+namespace Nighthawk {
+
+// An InputVariableSetter that sets the |requests_per_second| field in the CommandLineOptions proto.
+class RequestsPerSecondInputVariableSetter : public InputVariableSetter {
+public:
+  RequestsPerSecondInputVariableSetter(
+      const nighthawk::adaptive_load::RequestsPerSecondInputVariableSetterConfig& config);
```

Comment:
Maybe `RequestsPerSecondInputVariableSetter(
      const nighthawk::adaptive_load::RequestsPerSecondInputVariableSetterConfig& config) = default;` and drop the (empty) implementation from the .cc?

Reply:
It doesn't seem to like that, maybe it doesn't work when the constructor takes an arg that isn't this class.

......input_variable_setter_impl.h:13:86: error: only special member functions may be defaulted
      const nighthawk::adaptive_load::RequestsPerSecondInputVariableSetterConfig&) = default;

Reply:
Oh, sorry, yes that makes sense, I missed that. Sorry about the noise.